DuckDuckGo/AppDelegate.swift

```diff
@@ -293,7 +293,7 @@ import os.log
             bookmarksDatabase: bookmarksDatabase,
             appSettings: AppDependencyProvider.shared.appSettings,
             internalUserDecider: AppDependencyProvider.shared.internalUserDecider,
-            configurationStore: ConfigurationStore.shared,
+            configurationStore: ConfigurationStore(),
```

Comment:
Since this used to be a singleton (and I agree it shouldn't be any more), we can probably play it safe and put an instance of it on the `AppDependencyProvider`, just to make sure there are no assumptions that other parts of the codebase are making.